### PR TITLE
BaSP-TG-59: Sidebar component and style

### PR DESCRIPTION
**Pull request by HernanCazeneuve**
Captura de pantalla 2022-06-06 162642

### CODE REVIEW

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
The file vscode/settings. should not be there.

**Review by Gearyandres94 — DISMISSED** [review]
Great job!!!

**Review by Facund0Fernandez — APPROVED** [review]

File: src/Components/Layout/index.js [hunk 1]
```diff
@@ -21,24 +22,31 @@ function Layout() {
...
+            <Route exact path="/admins/form" component={AdminForm} />
+            <Route exact path="/super-admins" component={SuperAdmins} />
+            <Route exact path="/super-admins/form" component={SuperAdminForm} />
+            <Route exact path="/employees" component={Employees} />
+            <Route exact path="/employees/form" component={EmployeeForm} />
+            <Route exact path="/projects" component={Projects} />
+            <Route exact path="/projects/form" component={ProjectForm} />
+            <Route exact path="/time-sheets" component={TimeSheets} />
+            <Route exact path="/time-sheets/form" component={TimeSheetForm} />
+            <Route exact path="/Tasks" component={Tasks} />
```
Comment (AlejandroSantini) [inline src/Components/Layout/index.js:42]: You could lowercase the path

**Review by AlejandroSantini — APPROVED** [review]
Good job!!

**Review by Facund0Fernandez — DISMISSED** [review]

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
Why do you delete the content of the header????

**Review by GuidoCerioni — COMMENTED** [review]
Like Andy says, please return the header and header styles. We return it, and everything works well. If you left it deleted, when you merge, there is the chance you delete the real header.

**Pull request merged**